includes/class-base-widget.php

```diff
@@ -518,12 +610,33 @@ public function enqueue_scripts() {
 		$suffix = SCRIPT_DEBUG ? '' : '.min';
 
 		wp_enqueue_style( 'font-awesome', '//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css', [], '4.5.0' );
-		wp_enqueue_style( 'wpcw-admin', \Contact_Widgets::$assets_url . "css/admin{$suffix}.css", [ 'font-awesome' ], Plugin::$version );
-		wp_enqueue_script( 'wpcw-admin', \Contact_Widgets::$assets_url . "js/admin{$suffix}.js", [ 'jquery' ], Plugin::$version, true );
+		wp_enqueue_style( 'jquery-timepicker', Plugin::$assets_url . "css/jquery.timepicker{$suffix}.css", [ 'font-awesome' ], '1.11.9' );
```

Comment:
Well, since every widget shares the same JS file that expects the `wpcw_admin` global to always contain certain values, maybe the localize script data thing isn't necessary after all.

But the general idea of using inheritance for enqueuing other scripts and styles like `jquery-timepicker` and `font-awesome` still applies.